src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -274,6 +274,36 @@ static void resultFooter(RoutingContext ctx, Integer count, String diagnostic) {
         });
   }
 
+  Future<Void> getReportPackages(Vertx vertx, RoutingContext ctx) {
+    try {
```

Comment:
Try-catch is no longer needed as this is done by the add(RouterBuilder, ...) method.

Reply:
nice